docs/howto_quay_mirror.md

```diff
@@ -0,0 +1,294 @@
+> documentation for more information on how to install and configure the mirror registry.
+
+## Configure Certificates
+
+The mirror registry installer automatically generates an SSH key and an SSL
+certificate unless existing certificate files are specified from the command
+line using the `--ssh-key` and `--sslCert` arguments.
+
+The default keys and certificates are found in the `~/.ssh` and `quay-rootCA`
+subdirectory under the Quay root (i.e. `/var/lib/quay-root/quay-rootCA`).
+
+It is necessary to enable the SSL certificate trust on any host accessing the
+mirror registry. Copy the `rootCA.pem` file from the mirror registry to the
+target host at the `/etc/pki/ca-trust/source/anchors` directory and run the
+`update-ca-trust` command to enable the certificate in the system-wide trust
+store configuration.
+
+As an example, the following commands can be used to enable the certificate
+trust locally on the `microshift-quay` host.
+```
+MIRROR_ROOT="/var/lib/quay-root"
+
+sudo cp ${MIRROR_ROOT}/quay-rootCA/rootCA.pem /etc/pki/ca-trust/source/anchors/microshift-quay.pem
+sudo update-ca-trust
+```
+
+Test the connection to the mirror registry running on the `microshift-quay` host.
+The `curl` command should return no errors if the mirror registry service is up
+and running and the certificates are trusted.
+```
+MIRROR_HOST=microshift-quay:8443
+MIRROR_USER=microshift
+MIRROR_PASS=microshift
+
+curl -I -u ${MIRROR_USER}:${MIRROR_PASS} https://${MIRROR_HOST}
+```
+
+## Container Image List
+The list of the container image references used by a specific version of MicroShift
+is provided in the `release-<arch>.json` files that are part of the
+`microshift-release-info` RPM package.
+
+If the package is installed on a MicroShift host, the files can be accessed at
+the following location.
+```
+$ rpm -ql microshift-release-info
+/usr/share/microshift/release/release-aarch64.json
+/usr/share/microshift/release/release-x86_64.json
+```
+
+Alternatively, download and unpack the RPM package without installing it.
+```
+$ rpm2cpio microshift-release-info*.noarch.rpm | cpio -idmv
+./usr/share/microshift/release/release-aarch64.json
+./usr/share/microshift/release/release-x86_64.json
+```
+
+> Optionally use the `scripts/image-builder/download-rpms.sh` script for
+> downloading the released version of MicroShift RPM packages.
+
+The list of container images can be extracted into the `microshift-container-refs.txt`
+file using the following command.
+```
+RELEASE_FILE=/usr/share/microshift/release/release-$(uname -m).json
+jq -r '.images | .[]' ${RELEASE_FILE} > ~/microshift-container-refs.txt
+```
+
+> After the `microshift-container-refs.txt` file is created with the MicroShift
+> container image list, other user-specific image references can be appended to
+> the file before the mirroring procedure is run.
+
+## Mirror Container Images
```

Comment:
This section seems like the most important part, but it's hidden in this document that describes setting up VMs to act as the registry host. Most users will already have the registry, probably as part of their corporate infrastructure. What they really need is the steps for learning about which images to use and then how to configure MicroShift to look at that mirror.

Could we separate this into 2 parts, the user instructions for mirroring the images and consuming the mirror, then the developer/QE instructions for setting up a test environment with a registry in a VM?